polyply/tests/test_apply_modifications.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2024 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test that termini modifications are applied
+"""
+import logging
+
+import pytest
+import vermouth.forcefield
+import vermouth.ffinput
+from vermouth.molecule import Block
+from polyply.src.meta_molecule import MetaMolecule
+from polyply.src.apply_modifications import _get_protein_termini, apply_mod
+from polyply import TEST_DATA
+import polyply.src.ff_parser_sub
+import networkx as nx
+
+def example_meta_molecule(mods_dict = {}):
```

Comment:
I'd prefer if you import example_meta_molecule and then update the force-field with modifications like so:

```
from example_fixtures import example_meta_molecule
def test_function(example_meta_molecule, ...):
       update(example_meta_molecule.molecule.force_field, mods)
``` 
That way we don't have to add most of the code again